### ISSUE TITLE
Clean up the Torpedo Challenge Code

### ISSUE THREAD

**Issue by Grymessa** [uf-mil/SubjuGator#337]
Separate out the functions and make the centering movements more smooth. Code works as is but if we finish all the missions and perception files ahead of schedule we could improve this challenge. 

- [ ] Add exponential decay to movement of sub depending on how far off from center we are.
- [ ] Add active sonar integration to find distance from board (if we ever fix the Active Sonar)
- [ ] Fire torpedo command (this needs to be done before competition but can't be tested/used yet due to lack of valve box)
- [ ] Clean up functions

**Comment by kev-the-dev** [comment]
No longer relevant as 2019 involves new challenges
